### PR TITLE
Upgrade all dependencies.

### PR DESCRIPTION
**Pull request by DavDo**
**What was solved?**
I have upgraded all dependencies, and regenerated snapshots to align them with a new version of the paparazzi.

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to [**Logging from Android SDKs** → **Things to consider for newly added logs**](https://glia.atlassian.net/wiki/spaces/ENG/pages/3568861468/Logging+from+Android+SDKs#Things-to-consider-for-newly-added-logs) in Confluence for more information.

**Screenshots:**

### CODE REVIEW

File: gradle/libs.versions.toml
Comment (andrews-moc) [inline gradle/libs.versions.toml:33]: I expected to see only updated versions. However, `mediaExifInterfaceVersion` is a new one, it seems. Is this correct?

**Review by andrews-moc — COMMENTED** [review]
Looks like these two screenshots show the real differences: 
Screenshot 2024-10-21 at 15 10 53
Screenshot 2024-10-21 at 15 12 27

**Comment by DavDo** [comment]
@AndriiHorishniiMOC thanks for your research and fix! 👍

**Comment by AndriiHorishniiMOC** [comment]
The issue was in Paparazzi 1.3.4. Unfortunately, the next version has not yet been released. There is 1.3.5-SNAPSHOT, so we should decide whether to update now or wait until a stable release.

**Comment by DavDo** [comment]
> The issue was in Paparazzi `1.3.4`. Unfortunately, the next version has not yet been released. There is `1.3.5-SNAPSHOT`, so we should decide whether to update now or wait until a stable release.

Since it will stay in the feature branch and there are no visible problems with this snapshot build, I think we can go with the snapshot version.

**Comment by DavDo** [comment]
!squash

File: gradle/libs.versions.toml
Comment (DavDo) [reply gradle/libs.versions.toml:33]: Yes, because previously, the `android.core-ktx` library transitively contained this dependency, but in newer versions, they dropped it.

**Comment by DavDo** [comment]
> The issue was in Paparazzi `1.3.4`. Unfortunately, the next version has not yet been released. There is `1.3.5-SNAPSHOT`, so we should decide whether to update now or wait until a stable release.

Seems we will be unable to use this version. Sadly, it forces the Kotlin 2.0.20 version, and there is no way to change it back to 1.9.25.

**Comment by DavDo** [comment]
Paparazzi version 1.3.3 works well, so I'll leave this version.

**Comment by AndriiHorishniiMOC** [comment]
@DavDo, can we apply this plugin only to the snapshot build type?

**Comment by DavDo** [comment]
!squash

**Comment by DavDo** [comment]
> @DavDo, can we apply this plugin only to the snapshot build type?

Interesting Idea, let's try it.

**Comment by DavDo** [comment]
!squash

**Comment by DavDo** [comment]
> @DavDo, can we apply this plugin only to the snapshot build type?

This also breaks the Kotlin version of the project, so we will go with 1.3.3

File: build.gradle
Comment (AndriiHorishniiMOC) [inline build.gradle:28]: Why do we need it here? Is it a template?

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: build.gradle
Comment (DavDo) [reply build.gradle:28]: To use core sdk from the staging repository, which includes the latest SC V2 changes.

Comment (andrews-moc) [reply build.gradle:28]: Maybe it makes sense to add a TODO here to remove this line before release.

**Review by andrews-moc — APPROVED** [review]

File: widgetssdk/build.gradle
Comment (gugalo) [inline widgetssdk/build.gradle:84]: Please comment 'why' this configuration is needed.

**Review by gugalo — APPROVED** [review]

File: widgetssdk/build.gradle
Comment (DavDo) [reply widgetssdk/build.gradle:84]: This was needed with Paparazzi 1.3.4 to not fail while running on the local machine; I will recheck for the 1.3.3 version.

**Pull request merged**
